user/enterprise/worker-configuration.md

```diff
@@ -163,6 +163,65 @@ export TRAVIS_WORKER_DOCKER_BINDS="/tmp:/tmp:rw /var/log"
 A full list of options and mount modes is listed in the official
  [Docker documentation](https://docs.docker.com/storage/bind-mounts/).
 
+## Worker behind a HTTP(S) Proxy
+
+If you're using Travis CI Enterprise behind a HTTP(S) proxy, we've got you covered. Since travis-worker 4.6 it is possible to run builds behind a proxy.
+
+
+### How do I find out if I have the correct travis-worker version installed?
+
+#### Ubuntu 16.04+
+
+Connect to your worker machine via ssh and run:
+
+```
+$ sudo docker images | grep worker
+travisci/worker        v4.6.1                      ef7a3419050c        17 hours ago        44.7MB
+```
+
+#### Ubuntu 14.04
+
+Connect to your worker machine via ssh and run:
+
+```
+$ travis-worker -v
+travis-worker v=v4.6.1 rev=73392421d0ca807b83d4d459ad3dd484820fd181 d=2018-10-30T16:13:39+0000 go=go1.11.1
+```
+
+#### Upgrade travis-worker
+
+If you need to install a newer version of travis-worker, please follow the instructions in our [Updating your Travis CI Worker docs](/user/enterprise/upgrading/#updating-your-travis-ci-enterprise-worker).
+
+### Configuration
+
+On the worker machine, please open `/etc/default/travis-worker` in your editor and add these two lines:
+
+```
+export TRAVIS_WORKER_DOCKER_HTTP_PROXY="http://localhost:8080"
```

Comment:
This is their proxy URL right? Not a magic hard coded docker proxy url?

Reply:
Ahh, very good point. Let me change that right away.